elvis.config

```diff
@@ -3,7 +3,7 @@
     #{dirs => ["src", "test"],
       filter => "*.erl",
       rules => [
-        {elvis_text_style, line_length, #{limit => 80, skip_comments => false}},
+        {elvis_text_style, line_length, #{limit => 120, skip_comments => false}},
```

Comment:
It's more difficult to fit long lines to a editor screen with two panes.  I suggest leaving this at 80 or 90 chars at most.